src/hooks/settingsHooks/useGetSettings.ts

```diff
@@ -89,14 +90,7 @@ const convertfromBe = (backendSettings: BackendSiteSettings): SiteSettings => {
   }
 
   const rest = _(backendSettings)
-    .omit([
-      ...TOGGLED_VALUES,
-      "socialMediaContent",
-      "colors",
-      // These properties are extra and will lead to errors in validation
-      "resources_name",
-      "url",
-    ])
+    .omit([...TOGGLED_VALUES, "socialMediaContent", "colors"])
```

Comment:
double checking here, from the issue linked in the PR description, it seems like this PR only deals w/ `url`. could you help me understand why `resources_name` is being removed?

Reply:
The backend PR https://github.com/isomerpages/isomercms-backend/pull/490 removes the `resources_name` parameter from the settings API endpoint, so that would no longer be a need for this omission to be done on the frontend.

Reply:
ah gotcha